src/global_launch/config/globals.yaml

```diff
@@ -1,8 +1,15 @@
+# global parameters
 /**:
    ros__parameters:
       # Publishers' period (seconds): how often the publishers publish
       pub_period_sec: 0.5
 
+# local_pathfinding parameters
+mgp_main:
+   ros__parameters:
+      global_path_filepath: "/workspaces/sailbot_workspace/src/local_pathfinding/global_paths/mock_global_path.csv"
+      interval_spacing: 30.0
```

Comment:
Should `interval_spacing` be a double or float

Reply:
just a float